### PR TITLE
fix: new LogLevel enum is not exported properly #95

### LINKED ISSUE #95: bug: new LogLevel enum is not exported properly
In #94 I added/fixed the typings of the package.
As part of the process I added the LogLevel enum, which I exported in index.js.
When I tried to use the library, I realized that the npm package actually exports the the dist file, which is a minified version of node.js which doesn't export the new enum.
Also, it seems redundant to publish the src files if you expose a minified version.
@pi0 @Atinux I will prepare a quick fix not. Can you please give your opinion on the packaging of the src dir in the published package?

### PR DESCRIPTION
**Pull request by regevbr**
fix #95

### CODE REVIEW

**Comment by regevbr** [comment]
@pi0 @Atinux can you please CR this?
This is kind of "urgent" as the current release will break for who ever is using typescript :-(

File: package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "release": "standard-version && yarn build && git push --follow-tags && npm publish"
   },
   "files": [
-    "src",
```
Comment (pi0) [inline package.json:23]: Removing this is breaking change for 2.x (#51)

Comment (regevbr) [reply package.json:23]: So should I remove it?

Comment (regevbr) [reply package.json]: Never mind I removed it, the critical fix here is not related to it

**Comment by pi0** [comment]
Thanks for quick fix @regevbr :+1:  [v2.12.1](https://github.com/nuxt-contrib/consola/releases/tag/v2.12.1) released. Anyway will rewrite lib with typescript for next major so we don't have typing issues and also one dist in package.

**Comment by regevbr** [comment]
sounds good. Thanks for the quick response! I will try it now to see everything works properly

**Comment by regevbr** [comment]
I can confirm everything works great in Node :-)

**Pull request merged**